### PR TITLE
chore(root): publish modules

### PR DESCRIPTION
**Pull request by alebusse**
[WP-1463](https://bitgoinc.atlassian.net/browse/WP-1463)

Express: [9.61.0
](https://hub.docker.com/layers/bitgo/express/9.61.0/images/sha256-d34e81088c5544f85241b4054221c32af5cf7a2ed062f505baf7fff12169cd64?context=explore)

SDK:
 - @bitgo/abstract-cosmos@9.0.0
 - @bitgo/abstract-eth@19.0.0
 - @bitgo/abstract-utxo@7.0.0
 - @bitgo/account-lib@22.0.0
 - bitgo@37.0.0
 - @bitgo/express@9.61.0
 - @bitgo/key-card@0.22.0
 - @bitgo/sdk-api@1.43.0
 - @bitgo/sdk-coin-ada@2.36.0
 - @bitgo/sdk-coin-algo@1.26.0
 - @bitgo/sdk-coin-arbeth@19.0.0
 - @bitgo/sdk-coin-atom@11.0.0
 - @bitgo/sdk-coin-avaxc@2.28.0
 - @bitgo/sdk-coin-avaxp@3.22.0
 - @bitgo/sdk-coin-bch@1.6.1
 - @bitgo/sdk-coin-bcha@1.8.1
 - @bitgo/sdk-coin-bera@1.21.0
 - @bitgo/sdk-coin-bld@1.25.0
 - @bitgo/sdk-coin-bsc@21.0.0
 - @bitgo/sdk-coin-bsv@1.6.1
 - @bitgo/sdk-coin-btc@1.8.1
 - @bitgo/sdk-coin-btg@1.6.1
 - @bitgo/sdk-coin-celo@1.14.0
 - @bitgo/sdk-coin-coreum@19.0.0
 - @bitgo/sdk-coin-cspr@1.26.0
 - @bitgo/sdk-coin-dash@1.6.1
 - @bitgo/sdk-coin-doge@1.13.1
 - @bitgo/sdk-coin-dot@3.34.0
 - @bitgo/sdk-coin-eos@1.6.1
 - @bitgo/sdk-coin-etc@1.5.1
 - @bitgo/sdk-coin-eth@22.0.0
 - @bitgo/sdk-coin-eth2@1.5.1
 - @bitgo/sdk-coin-ethw@19.0.0
 - @bitgo/sdk-coin-hash@1.25.0
 - @bitgo/sdk-coin-hbar@1.12.0
 - @bitgo/sdk-coin-injective@1.25.0
 - @bitgo/sdk-coin-islm@1.21.0
 - @bitgo/sdk-coin-ltc@2.3.1
 - @bitgo/sdk-coin-near@1.7.1
 - @bitgo/sdk-coin-opeth@16.0.0
 - @bitgo/sdk-coin-osmo@1.27.0
 - @bitgo/sdk-coin-polygon@19.0.0
 - @bitgo/sdk-coin-rbtc@1.14.0
 - @bitgo/sdk-coin-sei@1.25.0
 - @bitgo/sdk-coin-sol@2.39.0
 - @bitgo/sdk-coin-stx@2.4.1
 - @bitgo/sdk-coin-sui@4.24.0
 - @bitgo/sdk-coin-tia@1.25.0
 - @bitgo/sdk-coin-ton@1.21.0
 - @bitgo/sdk-coin-trx@1.37.0
 - @bitgo/sdk-coin-xlm@2.24.0
 - @bitgo/sdk-coin-xrp@1.8.1
 - @bitgo/sdk-coin-xtz@1.7.1
 - @bitgo/sdk-coin-zec@1.6.1
 - @bitgo/sdk-coin-zeta@1.30.0
 - @bitgo/sdk-coin-zketh@1.9.0
 - @bitgo/sdk-core@26.0.0
 - @bitgo/sdk-lib-mpc@9.0.0
 - @bitgo/sdk-rpc-wrapper@1.23.0
 - @bitgo/sdk-test@8.0.0
 - @bitgo/sdk-unified-wallet@10.0.0
 - @bitgo/statics@47.0.0
 - @bitgo/utxo-bin@2.46.0
 - @bitgo/utxo-ord@1.20.0
 - @bitgo/web-demo@1.28.0

[WP-1463]: https://bitgoinc.atlassian.net/browse/WP-1463?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/account-lib/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,37 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [22.0.0](https://github.com/BitGo/BitGoJS/compare/@bitgo/account-lib@4.12.0...@bitgo/account-lib@22.0.0) (2024-02-19)
+
```
Comment (gianchandania) [inline modules/account-lib/CHANGELOG.md:7]: Why this change log is from 4.12.0 -> 22.0.0. Shouldn't this be from 21.0.0->22.0.0. The breaking change which is mentioned below, I think that was done long back, and is the major version update done considering that breaking change?

Comment (alebusse) [reply modules/account-lib/CHANGELOG.md:7]: This is generated automatically, and is probably missing the right tag for version 21.

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Review by margueriteblair — APPROVED** [review]

**Review by lcovar — APPROVED** [review]
